packages/main/test/specs/Input.mobile.spec.js

```diff
@@ -76,25 +76,26 @@ describe("Eventing", () => {
 		const input = await browser.$("#myInput");
 		const sExpectedSelectedKey = "Bg";
 		const sChangeText = "Bulgaria";
+		const sTypedText = "b";
 
 		await input.scrollIntoView();
 		await input.click();
 
 		const dialogInput = await input.shadow$("ui5-responsive-popover").$(".ui5-input-inner-phone");
-		await dialogInput.keys("b");
+		await dialogInput.keys(sTypedText);
 
 		const suggestionItem = await input.shadow$("ui5-responsive-popover").$$("ui5-li-suggestion-item")[2];
 		await suggestionItem.click();
 
 
-		const selectedKey = await browser.$("#myLabel").getText();
-		assert.strictEqual(selectedKey.split(" :: ")[1], sExpectedSelectedKey, "Selected key event property was correct");
+		const selectedKey = await browser.$("#myLabelSelectionChange").getText();
+		assert.strictEqual(selectedKey.split(" :: ")[1], sExpectedSelectedKey, "Selection change event property was correct");
 
 		const change = await browser.$("#myLabelChange").getText();
 		assert.strictEqual(change.split(" :: ")[1], sChangeText, "Change event property was correct");
 
 		const liveChange = await browser.$("#myLabelLiveChange").getText();
```

Comment:
we should replace 'liveChange' with 'input'

Reply:
Done